src/views/HelpAndKnowledge/Articles/_components/RelatedArticles/index.tsx

```diff
@@ -5,47 +5,64 @@ import { CardGroup, GridContainer } from '@trussworks/react-uswds';
 import classnames from 'classnames';
 
 import helpAndKnowledgeArticles, {
+  ArticleProps,
   ArticleTypeProps
 } from 'views/HelpAndKnowledge/Articles';
 import ArticleCard from 'views/HelpAndKnowledge/Articles/_components/ArticleCard';
 
 type RelatedArticlesProps = {
   className?: string;
   currentArticle: string;
-  viewAllLink?: boolean;
+  specificArticles?: string[];
   type?: ArticleTypeProps;
+  viewAllLink?: boolean;
 };
 
 const RelatedArticles = ({
   className,
   currentArticle,
-  viewAllLink,
-  type
+  specificArticles,
+  type,
+  viewAllLink
 }: RelatedArticlesProps) => {
   const { t } = useTranslation('helpAndKnowledge');
 
   // Filter to only the category tag type
-  let selectedArticles = type
+  let filteredArticles = type
     ? helpAndKnowledgeArticles.filter(article => article.type === type)
     : helpAndKnowledgeArticles;
 
   // If the only article is the current article, default to all articles
-  if (selectedArticles.length <= 1) {
-    selectedArticles = helpAndKnowledgeArticles;
+  if (filteredArticles.length <= 1) {
+    filteredArticles = helpAndKnowledgeArticles;
   }
 
   // Slice to first 3 of the relevant articles
-  selectedArticles = selectedArticles
+  filteredArticles = filteredArticles
     .filter(article => article.name !== currentArticle)
     .slice(0, 3);
 
+  // It first checks if `specificArticles` is defined, and if so, it maps over each article name to find the corresponding article object in `helpAndKnowledgeArticles`.
+  // It then filters out any undefined values and assigns the result to `articlesToShow`.
+  // If `specificArticles` is not defined, it assigns `filteredArticles` to `articlesToShow`.
+  const articlesToShow: ArticleProps[] =
+    specificArticles && specificArticles.length === 3
+      ? specificArticles
+          ?.map(articleName =>
+            helpAndKnowledgeArticles.find(
+              article => article.name === articleName
+            )
+          )
+          .filter((article): article is ArticleProps => article !== undefined) // Filter out undefined values
+      : filteredArticles;
```

Comment:
I think this could be a little clearer.  Maybe something like this.  Renaming the prop to `specificArticleNames` would be more accurate and allow us to use `specificArticles` when we map the actual articles

```
 specificArticleNames?: [string, string, string];
```

```
const specificArticles = specificArticleNames
    ?.map(articleName =>
      helpAndKnowledgeArticles.find(article => article.name === articleName)
    )
    .filter((article): article is ArticleProps => !!article); // Filter out undefined values

  const articlesToShow: ArticleProps[] = specificArticles || filteredArticles;
```